src/utils/notificationsLocalStorage.ts

```diff
@@ -20,20 +34,37 @@ export const setHandleOnCancel = () => {
 };
 
 export const setHandleOnCancelBanner = () => {
+  const accountIdLS = getLSAccountId();
+  const localStorageByAccountId = getNotificationLocalStorage();
+
   localStorage.setItem(
     NOTIFICATIONS_STORAGE,
     JSON.stringify({
-      ...getNotificationLocalStorage(),
+      ...getNotificationLocalStorageFull(),
+      [accountIdLS]: {
+        ...localStorageByAccountId,
+        bannerNotNowTS: Date.now(),
+      },
       bannerNotNowTS: Date.now(),
```

Comment:
why does this `bannerNotNowTS` attribute not exist within an `accountIdLS` scope?

Reply:
It's a need for now, we will remove it before the live release and the data will be stored by account only.